PWGDQ/Tasks/dqEfficiency.cxx

```diff
@@ -637,7 +638,7 @@ struct AnalysisSameEventPairing {
   }
 
   template <int TPairType, uint32_t TEventFillMap, uint32_t TEventMCFillMap, uint32_t TTrackFillMap, typename TEvent, typename TTracks1, typename TTracks2, typename TEventsMC, typename TTracksMC>
-  void runPairing(TEvent const& event, TTracks1 const& tracks1, TTracks2 const& tracks2, TEventsMC const& eventsMC, TTracksMC const& tracksMC)
+  void runPairing(TEvent const& event, TTracks1 const& tracks1, TTracks2 const& tracks2, TEventsMC const& eventsMC, TTracksMC const& tracksMC, bool flat = false)
```

Comment:
I suggest that the bool option to be made a task configurable. In this way we do not need to add additional process functions, and the "flat" option can be switched in all future process functions that might need these tables.